app/views/content/non-uk-teachers/why-train-to-teach-transcript.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Why train to teach in England?"
+description: |-
+  Full transcript for the 'Why train to teach in England?' video.
+---
+
+
+###  Why train to teach in England? 
+
+**Christobel Gondwe, Maths teacher:**
```

Comment:
As per the SKE transcript page - should we put the speaker information on a separate line for clarity? On the front end they are merged